src/devices/tuya.ts

```diff
@@ -8845,6 +8845,53 @@ const definitions: Definition[] = [
             }
         },
     },
+    {
+        fingerprint: [
+            {
+                modelID: 'TS011F',
+                manufacturerName: '_TZ3000_xeumnff9',
+            },
+        ],
+        model: 'ZMO-606-P2',
+        vendor: 'Zemismart',
+        description: 'Zemismart Smart 2 Poles Outlet (20A + 10A)',
+        extend: [
+            deviceEndpoints({'endpoints': {'1': 1, '2': 2}}),
+            identify(),
+            tuya.modernExtend.tuyaOnOff({indicatorMode: true, onOffCountdown: true, childLock: true, endpoints: ['1', '2']}),
+        ],
+        meta: {'multiEndpoint': true},
+    },
+    {
+        fingerprint: [
+            {
+                modelID: 'TS0003',
+                manufacturerName: '_TZ3000_aknpkt02',
+            },
+        ],
+        model: 'ZMO-606-S2',
+        vendor: 'Zemismart',
+        description: 'Zemismart Smart 2 Gangs Switch with Outlet',
+        extend: [
+            deviceEndpoints({'endpoints': {'1': 1, '2': 2, '3': 3}}),
```

Comment:
```suggestion
            deviceEndpoints({'endpoints': {'l1': 1, 'l2': 2, 'l3': 3}}),
```

And update `endpoints: ['l1', 'l2', 'l3']`

Make the same change for the `ZMO-606-P2`